cpp/data/zotero_cgi/qa.html

```diff
@@ -33,11 +33,11 @@ <h1>QA Settings</h1>
                 </tr>
             </thead>
             <tbody>
-                {LOOP tags, global_settings, journal_settings}
+                {LOOP tags, subfield_codes, global_settings, journal_settings}
```

Comment:
Also, the difference of regular_article settings to review_article settings must be added to this view!

Reply:
Done